ui/components/MesheryNotification.js

```diff
@@ -161,20 +163,30 @@ class MesheryNotification extends React.Component {
    * @param {number} type type of the event
    * @param {string} message message to be displayed
    */
-  notificationDispatcher(type, message) {
+  notificationDispatcher(type, message, operation_id) {
     const self = this;
     self.props.enqueueSnackbar(message, {
       variant : eventTypes[type]?.type,
       autoHideDuration : 5000,
       action : (key) => (
-        <IconButton
-          key="close"
-          aria-label="Close"
-          color="inherit"
-          onClick={() => self.props.closeSnackbar(key)}
-        >
-          <CloseIcon style={iconMedium} />
-        </IconButton>
+        <>
+          <IconButton
+            key="eye"
+            aria-label="eye"
+            color="inherit"
+            onClick={() => self.openEventInNotificationCenter(operation_id)}
+          >
+            <VisibilityIcon style={iconMedium} />
```

Comment:
@MdSahil-oss You're going to change to `MdInfoOutline`, right?

Reply:
@leecalcote Now, I've replaced `VisibilityIcon` with `InfoIcon` :)